modules/installation-and-upgrade/pages/containerized-proxy-wo-internet.adoc

```diff
@@ -0,0 +1,70 @@
+[[containerized-proxy-setup-without-internet]]
+= Containerized proxy deployment without an internet connection
+
+It is possible to deploy containerized {productname} Proxy in an environment without an internet connection.
+In such case, the images are copied to an internal registry, or saved to a tar file.
+
+This procedure allows re-use of the existing minion FQDN to create a configuration to start a containerized proxy.
+The user should first register the minion to suse manager server, and then re-use the same FQDN in a generation.
+
+
+Make sure the default port is set up to `8022` for virtualized proxy when creating configuration using [literal]`spacecmd`.
+The product that is installed in that machine will be SUSE Manager Proxy and will not have the needed packages.
+Changing the base channel to have the SUSE Manager client tools can lead to package conflicts and port conflicts.
+
+
+[IMPORTANT]
+====
+This procedure applies to {salt} proxies only.
+====
+
+[[from.suma.to.internal.reg.without.ssl]]
+== Copy images from suse manager registry to internal registry without SSL configured
+
+[NOTE]
+====
+If you want to copy the images to a registry with SSL, remove the property [literal]`--dest-tls-verify=false`.
+====
+
+. In a machine with access to "registry.suse.com" (CONFIRM: what machine - SUMA server?) install [literal]`skopeo` by using the command
```

Comment:
can be any machine, no need to be on suse manager machine

Reply:
@rjmateus - would this sentence work better: